content/ew-issue-z200-[2017-03-29].yaml

```diff
@@ -0,0 +1,146 @@
+issue: 200
+strapline: "Your weekly source of Ember news"
+content:
+
+    -
+        heading: "Headlines"
+        headlines:
+            -
+                title: "Announcing Glimmer.js"
+                description: ""
+                link: https://glimmerjs.com/
+            -
+                title: "Ember Community Survey 2017 Results"
+                description: ""
+                link: https://emberjs.com/ember-community-survey-2017/
+
+
+    # -
+    #     heading: "From our sponsors"
+    #     newsletterAd: true
+
+    # -
+    #     heading: "From our sponsors"
+    #     image: "https://github.com/ember-weekly/ember-weekly/raw/master/assets/sponsors/indeed-Logo_150x150.png"
+    #     advert: true
+    #     headlines:
+    #         -
+    #             title: "Let top tech companies apply to you on Indeed Prime​"
+    #             description: "Top software engineers are using Indeed Prime to land their dream job. Interested companies reach out with salary & equity upfront. Free for job seekers. Get $2k from Indeed Prime when you're hired."
+    #             advert: true
+    #             link: http://synd.co/2c1Ji0u
+    #             domain: indeed.com
+
+    -   
+        heading: "From our sponsors"
+        image: "https://github.com/ember-weekly/ember-weekly/raw/master/assets/sponsors/simplabs-gif.gif"
+        advert: true
+        headlines:
+            -
+                title: "Pro Ember.js: An advanced 2-day training event in Berlin"
+                description: "We'll tackle progressive web apps, modular architecture patterns, performance tuning, authentication & more!"
+                advert: true
+                link: http://ember-workshop.simplabs.com
+                domain: simplabs.com
+
+
+    -
+        heading: "Reading"
+        headlines:
+            -
+                title: "EmberConf 2017 Summary"
+                description: ""
+                link: https://github.com/poteto/emberconf-2017
+            -
+                title: "Rendering 10 Million Items with {{vertical-collection}} and Ember.js"
+                description: ""
+                link: https://build.addepar.com/engineering/rendering-10-million-items-with-vertical-collection/
+            -
+                title: "Lessons learned building a REST API theme for WordPress (with Ember)"
+                description: ""
+                link: https://blog.meetserenity.com/2017/03/lessons-learned-building-rest-api-theme-wordpress/
+            -
+                title: "Ember CLI Team Meeting Minutes - 2017-03-16"
+                description: ""
+                link: https://github.com/emberjs/core-notes/blob/master/ember-cli/2017-03/march-16.md
+            -
+                title: "Ember CLI Team Meeting Minutes - 2017-03-23"
+                description: ""
+                link: https://github.com/emberjs/core-notes/blob/master/ember-cli/2017-03/march-23.md
```

Comment:
"Ember CLI Team Meeting Minutes - 2017-03-16" & "Ember CLI Team Meeting Minutes - 2017-03-23" were [included in issue 199 but looks like they were miss-labelled](https://github.com/ember-weekly/newsletter/blob/master/content/ew-issue-z199-%5B2017-03-27%5D.yaml#L67)